thirdparty/build_gstreamer.sh

```diff
@@ -0,0 +1,8 @@
+cd gst-build
+./gst-worktree.py add gst-build-1.16 origin/1.16
+cd gst-build-1.16
+git checkout 1.16.2
+sudo /root/.local/bin/meson --prefix=${PWD}/outInstall builddir -Dpython=disabled
```

Comment:
1. why sudo is required for building it ? I can understand why it is required for installing
2. why do we need to address meson with it's full path if it is already installed ? worried that it may not work for every one in this manner @joiskash

Reply:
Sudo not required we used to install two versions of meson, one from apt-get and one from pip. We just want to make sure we are using the one from pip